examples/mod-data-import-converter-storage/entityTypeCollection.sample

```diff
@@ -0,0 +1,9 @@
+{
+  "entityTypes": [
+    "JOB_PROFILE",
```

Comment:
entityType != profileType

all entity types are declared into the story. Plz read description